### ISSUE TITLE
execution error: Error: SyntaxError: JSON Parse error: Unrecognized token '<' (-2700)

### ISSUE THREAD

**Issue by audiocommander** [alfredapp/snippet-transformer-workflow#1]
### Workflow version

2023.1

### Alfred version

5.1.4 (2195)

### macOS version

14.0 (23A344)

### Debugger output

```alfred_debugger
[11:22:52.656] Logging Started...
[11:22:58.085] Snippet Transformer[File Action] Processing complete
[11:22:58.093] Snippet Transformer[File Action] Passing output '/Users/mm/Downloads/Alfred Snippets/Alfred Snippets Backup/unix.alfredsnippets' to Arg and Vars
[11:22:58.095] Snippet Transformer[Arg and Vars] Processing complete
[11:22:58.096] Snippet Transformer[Arg and Vars] Passing output '/Users/mm/Downloads/Alfred Snippets/Alfred Snippets Backup/unix.alfredsnippets' to Run Script
[11:22:58.224] ERROR: Snippet Transformer[Run Script] /Users/mm/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/3A603FCA-65A2-4710-8420-300E781D7D35: execution error: Error: SyntaxError: JSON Parse error: Unrecognized token '<' (-2700)
[11:22:58.232] Snippet Transformer[Run Script] Processing complete
[11:22:58.234] Snippet Transformer[Run Script] Passing output '' to Conditional
[11:22:58.235] Snippet Transformer[Conditional] Processing complete
[11:22:58.236] Snippet Transformer[Conditional] Passing output '' to Run Script
[11:22:58.237] STDERR: Snippet Transformer[Run Script] /Users/mm/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/340AEDFE-5B0D-405C-873B-49E01610D4FD:1: no such file or directory:
[11:22:58.239] Snippet Transformer[Run Script] Processing complete
[11:22:58.240] Snippet Transformer[Run Script] Passing output '
' to Post Notification
[11:22:58.242] Snippet Transformer[Post Notification] Processing complete
[11:22:58.243] Snippet Transformer[Post Notification] Passing output '
' to Play Sound
[11:24:05.561] Snippet Transformer[File Action] Processing complete
[11:24:05.564] Snippet Transformer[File Action] Passing output '/Users/mm/Downloads/Alfred Snippets/Alfred Snippets Backup/C.alfredsnippets' to Arg and Vars
[11:24:05.565] Snippet Transformer[Arg and Vars] Processing complete
[11:24:05.567] Snippet Transformer[Arg and Vars] Passing output '/Users/mm/Downloads/Alfred Snippets/Alfred Snippets Backup/C.alfredsnippets' to Run Script
[11:24:05.686] ERROR: Snippet Transformer[Run Script] /Users/mm/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/3A603FCA-65A2-4710-8420-300E781D7D35: execution error: Error: SyntaxError: JSON Parse error: Unrecognized token '<' (-2700)
[11:24:05.688] Snippet Transformer[Run Script] Processing complete
[11:24:05.691] Snippet Transformer[Run Script] Passing output '' to Conditional
[11:24:05.694] Snippet Transformer[Conditional] Processing complete
[11:24:05.696] Snippet Transformer[Conditional] Passing output '' to Run Script
[11:24:05.755] STDERR: Snippet Transformer[Run Script] /Users/mm/Library/Caches/com.runningwithcrayons.Alfred/Workflow Scripts/340AEDFE-5B0D-405C-873B-49E01610D4FD:1: no such file or directory:
[11:24:05.758] Snippet Transformer[Run Script] Processing complete
[11:24:05.761] Snippet Transformer[Run Script] Passing output '
' to Post Notification
[11:24:05.763] Snippet Transformer[Post Notification] Processing complete
[11:24:05.765] Snippet Transformer[Post Notification] Passing output '
' to Play Sound
```

### More details

Hey, thanks for your workflow, it looks awesome, but unfortunately it doesn't work on my machine (Mac Book Air M2). I tested it with various snippets, also snippets that just contain plain text and no "<" character or special chars, but it always fails with the same error:

execution error: Error: SyntaxError: JSON Parse error: Unrecognized token '<' (-2700)

**Comment by vitorgalvao** [comment]
Please provide the file in question. You can zip it first so GitHub will allow it to be uploaded.

**Comment by audiocommander** [comment]
[C.zip](https://github.com/alfredapp/snippet-transformer-workflow/files/13404445/C.zip)
This one for example, I believe is because it contains quotes (").
I tried more of my snippet collections and there were indeed some that worked. It's probably because I have quite a lot of coding snippets with quotes or slashes. Thanks for checking!

**Comment by vitorgalvao** [comment]
Thank you, that helped. The issue wasn’t in the snippet itself, but I wasn’t ignoring the `info.plist` which includes more information on the collection.

Fixed in 2023.2.
